### ISSUE TITLE
Banlist

### ISSUE THREAD

**Issue by Flutscherino** [bra1n/judgebot#18]
!banlist format
would show the banned card in the format chosen.

**Comment by hgarus** [comment]
I gave this some thought when implementing !mtr, since it contains the banlists. Is printing a huge list of cards into a chat really useful? Personally I think this is better handled by a web page (!mtr will answer with a link to blogs.magicjudges.org/rules for requests of sections 6.3. to 6.7) or in reverse (printing the legality information with the card the way !card already does).

**Comment by bra1n** [comment]
I agree, I think it would be sufficient to return a link collection to the banlist for each format. Like:
```
Banlists:
Standard: http://...
Modern: http://....
etc.
```
